### ISSUE TITLE
More examples would be good - eg how to count planes per day as per your intro

### ISSUE THREAD

**Issue by gregpetersau** [danieldotnl/ha-measureit#54]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/danieldotnl/ha-measureit/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

It would be great if there were some examples on how to use it.
I cant seem to get the FlightRadar plane counting example you propose to work. I have tried state class of: measure, total, total increasing, but all i end up with is a value of Zero, or briefly the number of planes (as the input being flights entered zone is typically zero, with periodic pulses of 1,2 etc).

### Describe the solution you'd like

Assistance on how to perform the above, plus more examples on how to use the various options you have provided.

### Describe alternatives you've considered

tried every setting 

### Additional context

![image](https://github.com/danieldotnl/ha-measureit/assets/97826055/7882dc36-8490-494a-9706-008cb6726c37)
![image](https://github.com/danieldotnl/ha-measureit/assets/97826055/7df18e4a-c162-44ad-afc3-2e06400babb0)
![image](https://github.com/danieldotnl/ha-measureit/assets/97826055/9601b93b-c58e-46eb-8478-7b43196255a8)

**Comment by timbrd** [comment]
Same here. I haven't been able to count planes using flightradar integration. Examples how to use measureit would be great.

**Comment by bigramonk** [comment]
Agree, more examples would be nice.
I could not figure out a way to count the number of times my wine cellar door was opened in a day/week/month.

**Comment by Tobias-N7** [comment]
The concept of MeasureIt is great, but more examples would be really helpful. 

I'm trying to get time stats with full/empty home battery, but can't get it to work.

Condition template:

`{states('sensor.scb_battery_soc') | float  > 95}`
`{states('sensor.scb_battery_soc') | float  < 10}`

**Comment by danieldotnl** [comment]
For counting the planes during the night, I also created an automation that listens to the flightradar24_entry event and then updates a counter. Then MeasureIt uses that counter as a source sensor from 22:00-07:00h.

```yaml
alias: Flight counter
description: ""
trigger:
  - platform: event
    event_type: flightradar24_entry
condition: []
action:
  - service: counter.increment
    metadata: {}
    data: {}
    target:
      entity_id: counter.flights_in_area
mode: single
```
![flightrader_measureit1](https://github.com/danieldotnl/ha-measureit/assets/2983203/3b278e7a-7a52-4a58-b70d-532d268f3e2e)
![flightrader_measureit2](https://github.com/danieldotnl/ha-measureit/assets/2983203/dca753d8-b4d0-443c-bec1-0d300e37c69a)
![flightrader_measureit3](https://github.com/danieldotnl/ha-measureit/assets/2983203/07858cdb-0231-4eaa-928f-c2d9b754c577)

It would be very nice if MeasureIt can count events directly. I will add that to my roadmap!
@bigramonk that would solve your case as well, as opening the cellar door will trigger an event in home assistant.

**Comment by bigramonk** [comment]
>  @bigramonk that would solve your case as well, as opening the cellar door will trigger an event in home assistant.

I already have an automation triggered by the opening of my cellar which sends me a picture by Telegram. I'll add the counter, that should do the trick.
Thanks!

**Comment by AnthonyBe** [comment]
> It would be very nice if MeasureIt can count events directly. I will add that to my roadmap!

From my initial reading of description, I (and clearly others) thought that was what the integration actually did!
I really look forward to this capability being built-in to the integration!

**Comment by danieldotnl** [comment]
Now available in v0.5.4! Let me know your experiences!

**Comment by AnthonyBe** [comment]
> Now available in v0.5.4! Let me know your experiences!

Brilliant! It would be great if you could provide new step by step instructions for the example counting planes with Flight radar based on the new version of MeasureIt!
